src/openforms/config/service.py

```diff
@@ -0,0 +1,39 @@
+from dataclasses import dataclass
+
+from openforms.contrib.brk.checks import BRKValidatorCheck
+from openforms.forms.models.form import Form
+from openforms.plugins.exceptions import InvalidPluginConfiguration
+
+
+@dataclass
+class BrokenConfiguration:
+    config_name: str
+    exception_message: str | None
+
+
+def check_BRK_config_for_addressNl() -> str | None:
```

Comment:
```suggestion
def check_brk_config_for_addressnl() -> str | None:
```

I think it's better to stick with snake case, although `addressnl` feels a bit weird; so maybe `addressNL` is fine

Reply:
I think the best would be what you suggest (`addressNL`), in order to stick to the component's type as well.